### PR TITLE
feat: eng 871 dynamically update language suggestions

### PR DESCRIPTION
**Pull request by scottkelso**
# Description

### Issue

Need the languages in the suggestions dropdown to be more specific to the user and their search

[Link to Linear Todo](https://linear.app/jesus-film-project/issue/ENG-871/dynamically-update-language-suggestions)

### Solution

As long as there are languages available, set the two suggestions to equal
1. the most relevant language
2. the top two most relevant languages

# External Changes

Add a capitalizeFirstLetter utility function

# Additional information

Needs tested. Only prototyping.

### CODE REVIEW

**Review by GeronimoJohn — APPROVED** [review]
Not sure why linting is passing - but probs good to run `extract-translations` at some point in the feature branch

File: libs/journeys/ui/src/components/SearchBar/SearchDropdown/RefinementGroups/RefinementGroups.tsx [hunk 1]
```diff
@@ -44,11 +46,17 @@ export function RefinementGroups({
   const { refine: clearRefinements, canRefine: canClearRefinements } =
     useClearRefinements()
 
+  const { dispatch } = useSearchBar()
+  function unselectAllLanguages(): void {
+    clearRefinements()
+    dispatch({ type: 'RemoveAllLanguageContinents' })
+  }
```
Comment (GeronimoJohn) [inline libs/journeys/ui/src/components/SearchBar/SearchDropdown/RefinementGroups/RefinementGroups.tsx:53]: can we update name to be `handleClick` please

File: libs/journeys/ui/src/libs/algolia/SearchBarProvider/SearchBarProvider.tsx [hunk 2]
```diff
@@ -110,6 +115,13 @@ function removeLanguageContinents(
   }
 }
 
+function removeAllLanguageContinents(state: SearchBarState): SearchBarState {
+  return {
+    ...state,
+    continentLanguages: {}
+  }
+}
```
Comment (GeronimoJohn) [inline libs/journeys/ui/src/libs/algolia/SearchBarProvider/SearchBarProvider.tsx:123]: can we add a test for this please

File: libs/journeys/ui/src/components/SearchBar/SearchDropdown/RefinementGroups/RefinementGroups.tsx (hunk 1 above)
Comment (scottkelso) [reply libs/journeys/ui/src/components/SearchBar/SearchDropdown/RefinementGroups/RefinementGroups.tsx:53]: 😥

**Review by GeronimoJohn — APPROVED** [review]

**Pull request merged**